packages/visually-hidden/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Render text that is announced to screen readers but visually hidden.",
   "main": "index.js",
   "module": "es/index.js",
+  "typings": "typings/index.d.ts",
```

Comment:
I think to remain consistent with my PRs (unless anyone feels strongly otherwise) let's move this file into the root of the package :+1:

Reply:
Actually I did that first but than I thought that another folder would be a good idea 😅 But I agree it is better to keep consistence 👌

Reply:
Cool, well as soon as you make that change I will approve this for you :+1: 🙂